src/main/java/executor/service/SecurityConfig.java

```diff
@@ -0,0 +1,41 @@
+package executor.service;
```

Comment:
I recommend placing this class in **package executor.service.config;**

Reply:
It's better do on the local merge if there are conflict, in this branch no package  executor.service.config.